posts/Yoann/2023-06-26-hibernate-search-6-2-0-CR1.adoc

```diff
@@ -0,0 +1,113 @@
+an upgrade to Hibernate ORM 5.6.15.Final,
+an upgrade of `-orm6` artifacts to Hibernate ORM 6.2.5.Final,
+and other bugfixes and improvements.
+
+== What's new
+
+[NOTE]
+====
+For a summary of all new features and improvements since 6.1,
+head to https://hibernate.org/search/releases/6.2/#whats-new[the dedicated page on hibernate.org].
+====
+
+=== Dependency upgrades
+
+[[orm-version]]
+Hibernate ORM (link:{hsearch-jira-url-prefix}/HSEARCH-4880[HSEARCH-4880]/link:{hsearch-jira-url-prefix}/HSEARCH-4875[HSEARCH-4875]/link:{hsearch-jira-url-prefix}/HSEARCH-4869[HSEARCH-4869])::
+Hibernate Search now depends on Hibernate ORM 5.6.15.Final for its main artifacts,
+and 6.2.5.Final for `-orm6` artifacts.
+Hibernate ORM 6.0 and 6.1 are no longer considered compatible.
+[[elasticsearch-version]]
+Elasticsearch (link:{hsearch-jira-url-prefix}/HSEARCH-4857[HSEARCH-4857])::
+The Elasticsearch backend now works with Elasticsearch 8.8
+as well as other versions that were already compatible.
+[[opensearch-version]]
+OpenSearch (link:{hsearch-jira-url-prefix}/HSEARCH-4870[HSEARCH-4870])::
+The Elasticsearch backend now works with OpenSearch 2.8
+as well as other versions that were already compatible.
+[[others-version]]
+Others::
+* link:{hsearch-jira-url-prefix}/HSEARCH-4871[HSEARCH-4871]: Upgrade to Elasticsearch **client** 8.8.1
+
+[[indexedembedded-excludepaths]]
+=== `@IndexedEmbedded(excludePaths = ...)`
+
+With link:{hsearch-jira-url-prefix}/HSEARCH-1182[HSEARCH-1182],
+the `@IndexedEmbedded` annotation now exposes an `excludePaths` attribute,
+allowing the inclusion of all paths with only a few selectively excluded,
+as opposed to the previous approach of selectively including paths with `includePaths`.
+
+See link:{hsearch-doc-url-prefix}#mapping-indexedembedded-filtering[this section of the reference documentation]
+for more information.
+
+[[objectprojection-filters]]
+=== `@ObjectProjection(includeDepth = ...)`/`@ObjectProjection(includePaths = ...)`/`@ObjectProjection(excludePaths = ...)`
+
+With link:{hsearch-jira-url-prefix}/HSEARCH-4725[HSEARCH-4725],
+the `@ObjectProjection` annotation now exposes an `includeDepth`/`includePaths`/`excludePaths` attributes,
+which allows in particular breaking cycles of nested object projections.
+
+See link:{hsearch-doc-url-prefix}#search-dsl-projection-object-mapping-filters[this section of the reference documentation]
+for more information.
+
+[[other-changes]]
+=== Other improvements and bug fixes
+
+* link:{hsearch-jira-url-prefix}/HSEARCH-4616[HSEARCH-4616]:
+For clarity, "automatic indexing" was renamed to something more explicit throughout Hibernate Search and its documentation.
+As a result:
+** `hibernate.search.automatic_indexing.enabled` is now deprecated in favor of `hibernate.search.indexing.listeners.enabled`
+** `hibernate.search.automatic_indexing.synchronization.strategy` is now deprecated in favor of `hibernate.search.indexing.plan.synchronization.strategy`
```

Comment:
We just add it here for completeness right ( I think we've released this one in a previous version)?

Reply:
Yep, this felt relevant.